sdk/examples/intkey_rust_cli/src/getstate.rs

```diff
@@ -0,0 +1,119 @@
+use hyper;
+use hyper::Method;
+use hyper::client::{Client, Request};
+use futures::{future, Future};
+use futures::Stream;
+use tokio_core;
+
+use serde_json::{Value, Map, from_str };
+use serde_cbor::{ from_slice };
+use base64::{ decode };
+
+use errors::IntkeyError;
+
+use crypto::sha2::Sha512;
+use crypto::digest::Digest;
+
+pub fn storage_addr_from_name(_name: &str) -> String {
+    let mut sha = Sha512::new();
+    let name_bytes = _name.as_bytes();
+    sha.input(name_bytes);
+    let result_string = sha.result_str();
+    let hashed_name_bytes = result_string.as_bytes();
+    let as_string = String::from_utf8_lossy(&hashed_name_bytes[64..]).into_owned();
+    as_string
+}
+
+
+pub fn decode_and_format_response(x: &Value) -> Result<(), IntkeyError> {
+    //Acts on individual key/value pairs.
+    let base64_decoded = x.get(String::from("data"))?
+                                           .as_str()?;
+               
+ 
+    let cbor_string = decode(base64_decoded)?;
+
+    let kv_map: Map<String, Value> = from_slice(&cbor_string[0..])?;
+    let mut key_iter = kv_map.keys();
+    let key_name = key_iter.next()?;
+    let value = kv_map.get(key_name)?
+                      .as_u64()?;
+
+    println!("  {} -> {}", key_name, value);
+    Ok(())
+
+}
+
+pub fn get_state(_url: Option<&str>, _key_name: Option<&str>) -> Result<(), IntkeyError> {
+
+
+    let key_address = match _key_name {
+        Some(v) => storage_addr_from_name(v),
+        None => String::from(""),
+    };
+
+    let get_url = match _url {
+        Some(v) => format!("{}{}{}", v, "/state?address=1cf126", key_address),
+        None => format!("http://127.0.0.1:8008/state?address=1cf126{}", key_address),
```

Comment:
Add a constant for this.  Something like:
const STRING: &'static DEFAULT_URL = "http://127.0.0.1:8008";
This will make it easy to change, for example, for use with Docker containers.